src/unit_tests/shared/test_enrollment_op.c

```diff
@@ -337,6 +359,31 @@ void test_w_enrollment_concat_group(void **state) {
     w_enrollment_concat_group(buf, group);
     assert_string_equal(buf, " G:'EXAMPLE_GROUP'");
 }
+/**********************************************/
```

Comment:
Code style: Separate the tests with a new line previous to this header.
Same as in line 345.

Reply:
Done!

Reply:
Added some more style fixes